frontend/src/pages/team/Billing.vue

```diff
@@ -1,4 +1,7 @@
 <template>
+    <div v-if="subscription?.customer?.balance < 0" class="ff-banner ff-banner-warning mb-3">
```

Comment:
Do we have other flavours of banner to consider? The red of warning doesn't feel appropriate for this informational message.

Reply:
This was intended very much as a temporary banner before the #1521 improvements, but very fair point, I'll pick a lighter color.

Reply:
Color toned down to a softer blue, ![Screenshot 2023-01-12 at 16 09 13](https://user-images.githubusercontent.com/507155/212103298-84133838-db7a-4f6b-8aca-39db24f79324.png), will iterate on the best way to show the credit with Joe in #1521.